packages/terra-wallet-provider/lib/TerraWalletProvider.ts

```diff
@@ -202,7 +206,17 @@ export default class TerraWalletProvider extends WalletProvider {
       const gasPrice = data.fee as any
       const gasLimit = 800000
 
-      const fee = ceil(new BigNumber(gasLimit).times(gasPrice).toNumber())
+      let taxFee
+      if (this._stableFee) {
+        taxFee = new BigNumber((await this.getTaxFees(value.toNumber() / 1_000_000, this._feeAsset, false)) * 1_000_000)
```

Comment:
can we please split this statement into multiple lines with variable names? it will make the code more readable.